plotting/plot_handler.py

```diff
@@ -84,20 +84,23 @@ def get_plot_file(self):
         :return: (str) local path to downloaded files OR None if no files found
         """
         _existing_plot_files = self._check_for_plot_files()
+        local_plot_paths = []
         if _existing_plot_files:
-            # Generate paths to data on server and destination on local machine
-            _server_path = self.server_dir + _existing_plot_files[0]
-            _local_path = os.path.join(self.static_graph_dir, _existing_plot_files[0])
-
             client = SFTPClient()
-            try:
-                client.retrieve(server_file_path=_server_path,
-                                local_file_path=_local_path,
-                                override=True)
-                LOGGER.info('File %s found and saved to %s', _server_path, _local_path)
-            except RuntimeError:
-                LOGGER.error("file does not exist")
-                return None
-            return f'/static/graphs/{_existing_plot_files[0]}'  # shortcut to static dir
+            for plot_file in _existing_plot_files:
+                # Generate paths to data on server and destination on local machine
+                _server_path = self.server_dir + plot_file
+                _local_path = os.path.join(self.static_graph_dir, plot_file)
+
+                try:
+                    client.retrieve(server_file_path=_server_path,
+                                    local_file_path=_local_path,
+                                    override=True)
+                    LOGGER.info('File %s found and saved to %s', _server_path, _local_path)
+                except RuntimeError:
+                    LOGGER.error("file does not exist")
```

Comment:
Should we add the file and location that can't be found in this log for more context?